kafka/producer/kafka.py

```diff
@@ -253,7 +256,7 @@ class KafkaProducer(object):
         'api_version': 'auto',
     }
 
-    def __init__(self, **configs):
+    def __init__(self, client=None, **configs):
```

Comment:
same as above -- you should just add `'client': None` to the class DEFAULT_CONFIG dict